### PR TITLE
Added demon to remove stale data.

### PR DESCRIPTION
**Pull request by Pilipets**
Signed-off-by: Hlib Pylypets <pilipets.gleb@gmail.com>

### CODE REVIEW

**Comment by Pilipets** [comment]
The question to think is do we need to check the file system for deletion of the user related files as well?

File: web-service/src/stale_demon/stale_demon.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+)
+
+var (
+	db          containers.ClientContainer
+	runPeriod   int64
+	errorLogger *log.Logger
+	debugLogger *log.Logger
+)
+
+func InitializeDemon(container containers.ClientContainer, period int64) {
```
Comment (gevorgyana) [inline web-service/src/stale_demon/stale_demon.go:19]: It is called Daemon.

**Comment by gevorgyana** [comment]
> The question to think is do we need to check the file system for deletion of the user related files as well?

I think no. We should never do that as the design specifies that we use S3.

File: web-service/src/stale_demon/stale_demon.go (hunk 1 above)
Comment (Pilipets) [reply web-service/src/stale_demon/stale_demon.go:19]: демон
demon, daemon, fiend, incubus

Comment (Pilipets) [reply web-service/src/stale_demon/stale_demon.go:19]: Ok, I googled that it should be daemon https://en.wikipedia.org/wiki/Daemon_(computing)

**Review by gevorgyana — APPROVED** [review]

**Pull request merged**